src/lib/api-client.js

```diff
@@ -184,3 +202,14 @@ export function getModelsForNewItem() {
         hashHistory.push('/items/new');
     });
 }
+export function deleteItem(address){
+    return del('item', {
+        itemAddress: address
+    }).then(data => {
+        Dispatcher.handleAction('ITEMS_RECEIVED', data);
```

Comment:
It does not need to differentiate. I like having two different actions, but I agree it's out of scope since I don't need ITEM_DELETED for the requirements of this story.